.devcontainer/devcontainer.json

```diff
@@ -41,7 +51,7 @@
     }
   }
   // Use 'forwardPorts' to make a list of ports inside the container available locally.
-  // "forwardPorts": [3000, 5432],
+  // "forwardPorts": [3000, 5434],
```

Comment:
Why change this?

Reply:
I was doing tests, will revert it to minimise noise.